src/index.ts

```diff
@@ -496,7 +497,7 @@ export class Loader {
       key: this.apiKey,
       channel: this.channel,
       client: this.client,
-      libraries: this.libraries,
+      libraries: this.libraries.length && this.libraries,
```

Comment:
I can not comment on l 536/537 as not touched by this PR but do we need async/await there ?